src/core/service/RestaurantsService.js

```diff
@@ -3,48 +3,33 @@ export default class RestaurantsService {
         this.httpRequest = httpRequest;
     }
 
-    getAllCatgories() {
-        return Promise.resolve(["Burger", "Rybka"]);
+    uniqBy(a, key) {
```

Comment:
Na co to? Jeżeli kategorie się duplikują to bug na backendzie jest. Nic nie filtrujemy